meta-lmp-base/classes/fip-utils.bbclass

```diff
@@ -3,10 +3,6 @@ DEPENDS += "tf-a-tools-native"
 # Configure new package to provide fiptool wrapper for SDK usage
 PACKAGES =+ "${FIPTOOL_WRAPPER}"
```

Comment:
I think we can drop this fiptool wrapper too because it is only used by the native sdk